cmake/kokkos_tpls.cmake

```
@@ -85,6 +85,11 @@ ENDIF()
 KOKKOS_IMPORT_TPL(ROCM INTERFACE)
 KOKKOS_IMPORT_TPL(LIBQUADMATH)
 
+IF (KOKKOS_ENABLE_IMPL_DESUL_ATOMICS AND KOKKOS_ENABLE_DESUL_ATOMICS_EXTERNAL)
```

```suggestion
IF(Kokkos_ENABLE_IMPL_DESUL_ATOMICS AND Kokkos_ENABLE_DESUL_ATOMICS_EXTERNAL)
```
You have not addressed my previous comment about the case of these variables https://github.com/kokkos/kokkos/pull/5053#discussion_r886123158

---

We are using `Kokkos_ENABLE_IMPL_DESUL*` etc. now.